modules/elb.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: module
+title: Elastic Load Balancing
+image: assets/img/ec2.png
+description: Create and manage Elastic Load Balancers
+---
+Overview
+--------
+Cumulus makes configuring Elastic Load Balancers simpler. Easily configure load balancers, listeners, and policies. Read the following sections to learn about configuring your load balancers with Cumulus. Example configuration can be found in the [Cumulus repo](https://github.com/lucidsoftware/cumulus).
+
+Load Balancer Definition
+-----------------------
+
+Each load balancer is defined in its own file where the file name is the name of the load balancer. These files are located in a [configurable](#configuration) folder. Load balancers are JSON objects with the following attributes:
+
+* `listeners` - the configuration for the listeners on the load balancer. Listeners can be defined using templates and/or inline (see [listeners](#listeners). At least one listener between `includes` and `inlines` must be present for a valid load balancer configuration. Has the following properties:
```

Comment:
A valid load balancer configuration has at least one listener in either `includes` or `inlines`